lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.ts

```diff
@@ -107,16 +107,16 @@ export class SearchDateRangeComponent implements SearchWidget, OnInit, OnDestroy
 
         if (this.settings?.maxDate) {
             if (this.settings.maxDate === 'today') {
-                this.maxDate = this.dateAdapter.today().endOf('day');
+                this.maxDate = endOfToday();
             } else {
-                this.maxDate = moment(this.settings.maxDate).endOf('day');
+                this.maxDate = endOfDay(new Date(this.settings.maxDate));
             }
         }
 
         if (this.startValue) {
             const splitValue = this.startValue.split('||');
-            const fromValue = this.dateAdapter.parse(splitValue[0], this.datePickerFormat);
-            const toValue = this.dateAdapter.parse(splitValue[1], this.datePickerFormat);
+            const fromValue = this.dateAdapter.parse(splitValue[0], DateFnsUtils.convertMomentToDateFnsFormat(this.datePickerFormat));
```

Comment:
not needed, you already converted that in the ngOnInit

```
this.datePickerFormat = this.settings?.dateFormat ? DateFnsUtils.convertMomentToDateFnsFormat(this.settings.dateFormat) :
        DateFnsUtils.convertMomentToDateFnsFormat(DEFAULT_FORMAT_DATE);
```

Reply:
Done